module/rdpClientCon.c

```diff
@@ -971,14 +971,14 @@ rdpClientConProcessMsgClientInfo(rdpPtr dev, rdpClientCon *clientCon)
     {
         LLOGLN(0, ("  client can not do new(color) cursor"));
     }
-    if (clientCon->client_info.monitorCount > 0)
+    if (clientCon->client_info.display_sizes.monitorCount > 0)
```

Comment:
Can you keep compatibility with older versions of xrdp? Probably using some macros and check `CLIENT_INFO_CURRENT_VERSION`.

This is not always necessary but I want that at this time.